CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v6.1.1 (04/12/2021)
+
+#### Bug fixes and enhancements:
+
+* [NSTK-2256] Add missing toleration to db pods
+* [NSTK-2259] [Support DB pods StatefulSet node-affinity and topology](https://github.com/purestorage/pso-csi/blob/master/docs/db-topology.md)
+* [#90](https://github.com/purestorage/pso-csi/issues/90) Enhance PSO log debug level
```

Comment:
[nit] I think just putting #90 by itself will make it a link, but I'm fine leaving it as a link like this.